src/literals.py

```diff
@@ -26,6 +26,8 @@
 ELECTION_PORT = 3888
 JMX_PORT = 9998
 METRICS_PROVIDER_PORT = 7000
+USER = 584788
```

Comment:
I would probably [link the docs](https://snapcraft.io/docs/system-usernames), or just have a comment here. It's quite obscure otherwise.

Reply:
Done in fe7997cbc9c4985300600bd7e1c2c3b987146921